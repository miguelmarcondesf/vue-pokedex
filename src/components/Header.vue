<template>
  <v-toolbar dark color="red">
    <v-toolbar-side-icon></v-toolbar-side-icon>

    <v-toolbar-title class="toolbal-logo">
      <router-link to="/">Pok√©Dex Vue</router-link>
    </v-toolbar-title>

    <v-spacer></v-spacer>

    <v-btn icon>
      <v-icon>search</v-icon>
    </v-btn>

  </v-toolbar>
</template>

<script>
export default {

}
</script>

<style lang="scss" scoped>
  .toolbal-logo {
    a  {
      color: white;
      text-decoration: none;
    }
  }
</style>
