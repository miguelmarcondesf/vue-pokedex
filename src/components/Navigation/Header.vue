<template>
  <v-toolbar dark color="red">
    <v-toolbar-side-icon></v-toolbar-side-icon>

    <v-toolbar-title class="toolbal-logo">
      <router-link to="/">Pok√©dex Vue</router-link>
    </v-toolbar-title>

    <v-spacer></v-spacer>

      <!-- <v-text-field
        class="search-input"
        label="Search"
        prepend-inner-icon="search"
      ></v-text-field> -->

  </v-toolbar>
</template>

<script>
export default {

}
</script>

<style lang="scss" scoped>
  .toolbal-logo {
    a  {
      color: white;
      text-decoration: none;
    }
  }

  .search-input {
  }
</style>
