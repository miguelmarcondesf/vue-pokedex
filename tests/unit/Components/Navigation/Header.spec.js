import { shallowMount } from '@vue/test-utils'
import Header from '@/components/Navigation/Header.vue'
import Vue from 'vue'
import Vuetify from 'vuetify'

Vue.use(Vuetify)

describe('Header', () => {
  it('is a Vue instance', () => {
    const wrapper = shallowMount(Header, {
      stubs: ['router-link', 'router-view', 'store']
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('should render content correctly', () => {
    const wrapper = shallowMount(Header, {
      stubs: ['router-link', 'router-view']
    })

    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
