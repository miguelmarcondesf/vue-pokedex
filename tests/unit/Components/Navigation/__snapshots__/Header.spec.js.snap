// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render content correctly 1`] = `
<v-toolbar-stub
  color="red"
  dark="true"
  scrollthreshold="300"
>
  <v-toolbar-side-icon-stub />
   
  <v-toolbar-title-stub
    class="toolbal-logo"
  >
    <router-link-stub
      to="/"
    >
      Pok√©dex Vue
    </router-link-stub>
  </v-toolbar-title-stub>
   
  <v-spacer-stub />
</v-toolbar-stub>
`;
